### DISCUSSION TITLE
Elastic traditional Cluster during install

### DISCUSSION THREAD

**Discussion by rosswakelin** [Security-Onion-Solutions/securityonion#3411] [Unsupported Versions]
During the install of the "Master" node configuration, you get asked if you want to use "traditional" elastic cluster or standalone.  When I selected traditional cluster, the install went ahead a finished, but then when rebooted the Elasticsearch instance would not start up.  The logs complained about not finding a master-capable node.  I changed the global.sls file to switch to the standaone instance, and once rebooted the Elasticsearc database started up, but nothing in SO worked, cos none of the SO specific stuff had been loaded into Elastic and Kibana (cos the database was not available/failed to start during the install process).

Question - is there an issue with using clustered Elasticsearch nodes? Do you have to use a number of standalone nodes as the way of scaling storage?

**Comment by weslambert** [comment]
Did you try restarting again/rebooting before making changes to the cluster type? If IIRC, it should cut over once another node is added.  If you have any more detail you'd like to share, or can replicate this, please let us know.
